lldb/test/API/lang/swift/cxx_interop/backward/test-format-swift-types-in-cxx/main.cpp

```diff
@@ -1,10 +1,62 @@
 #include "swift-types.h"
 
-int main() {
-  using namespace a;
+using namespace a;
+
+int testSwiftClass() {
   auto swiftClass = returnSwiftClass();
-  auto swiftSublass = returnSwiftSubclassAsClass();
+  return 0; // Break here for class
+}
+
+int testSwiftSubclass() {
+  auto swiftSublass = returnSwiftSubclass();
+  return 0; // Break here for subclass
```

Comment:
You might want to `print("Break here ...")` to make sure there is actual code on this line.